woudc_data_registry/models.py

```diff
@@ -602,15 +616,15 @@ def __init__(self, ecsv):
             self.data_generation_scientific_authority = \
                 ecsv.extcsv['DATA_GENERATION']['ScientificAuthority']
 
-        self.platform_type = ecsv.extcsv['PLATFORM']['Type']
-        self.platform_name = ecsv.extcsv['PLATFORM']['Name']
-        self.platform_country = ecsv.extcsv['PLATFORM']['Country']
-        self.platform_gaw_id = ecsv.extcsv['PLATFORM'].get('GAW_ID', None)
+        self._platform_type = ecsv.extcsv['PLATFORM']['Type']
```

Comment:
Why are we prefixing all properties with `_` ?

Reply:
Any field with an underscore is also an `@property` down below. Externally you can and should access the station type using the `platform_type` property, so the interface is the same. Internally it either comes from `_platform_type` (if the object was made from an ecsv file and __init__ is called) or from a relationship reference (if the object comes straight from PSQL). That way you can reference `platform_type` no matter which source the object comes from.